FlubberView/FlubberView.swift

```diff
@@ -110,22 +111,27 @@ public extension FlubberView {
         for v in subviews {
             let initialPoint = nodeCenterCoordinates.object(forKey: v)?.cgPointValue ??
                 CGPoint(x: v.frame.midX, y: v.frame.midY)
-            let elasticity = magnitude.elasticity
-            let snapBehavior = UISnapBehavior(item: v, snapTo: initialPoint)
 
+            let bounceBehavior = bounce(for: v)
+            bounceBehavior.damping = damping
+            bounceBehavior.frequency = frequency
 
-            snapBehavior.damping = damping
-
-            let oldBehavior = behaviors.object(forKey: v)
-            behaviors.setObject(snapBehavior, forKey: v)
+            let oldBehavior = attachBehaviors.object(forKey: v)
+            attachBehaviors.setObject(bounceBehavior, forKey: v)
 
             if let behavior = oldBehavior {
                 mainAnimator.removeBehavior(behavior)
             }
 
-            mainAnimator.addBehavior(snapBehavior)
-            v.center = CGPoint(x: v.center.x <~> elasticity, y: v.center.y <~> elasticity)
+            mainAnimator.addBehavior(bounceBehavior)
+            v.center = CGPoint(x: v.center.x <~> magnitude.elasticity, y: v.center.y <~> magnitude.elasticity)
             mainAnimator.updateItem(usingCurrentState: v)
+            DispatchQueue.main.asyncAfter(deadline: DispatchTime.now() + duration, execute: {
```

Comment:
#pp debounce this so that if the flubber view is tapped repeatedly, only the last snap back fires